scripts/olean_comparison.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+
+oleansDir=.lake/build/lib/Mathlib
+# should be master
```

Comment:
Is there anything preventing us from pointing this at master and then doing a few test runs with existing branches?

Reply:
No, my idea is to test this now pointing it at the current branch, but, before merging the PR, point it at master, where the data will be available.

Reply:
I think running tests with its final configuration is a good idea before merging.

Reply:
What kind of tests would you like to see?  I could implement a version of this script in Bryan's fork of mathlib, where I could push this into master and then run it, on demand, on various commits to see the relative outcomes.  However, it would still be a CI on a different repository, even though it could compute information about the "real" mathlib4.

Reply:
Or, I could convert this into a script that triggered by a comment (that would download master's `olean`s) and run the comparison.  That would probably still need to be merged into master to work, but would be much safer, since it would not be on the main CI build, so any bugs would just affect the comment workflow.